components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -489,7 +492,10 @@ interface KeyringService {
   IsLocked() => (bool isLocked);
 
   // Adds an account to the keyring.
-  AddAccount(string accountName, CoinType coin ) => (bool success);
+  AddAccount(string accountName, CoinType coin) => (bool success);
```

Comment:
We should also put a comment for this API saying use `AddFilecoinAccount` for Filecoin. Same rule applies to `GetSelectedAccount` and `ImportAccount`